daemon/servers.go

```diff
@@ -126,6 +126,11 @@ func getServers(
 			serverGroup,
 			obfuscated,
 		)
+
+		// remove all DIP servers from the list if the search wasn't made for a server name and there is more then 1 server found
+		if err == nil && serverTag.Action != core.ServerByName && len(ret) > 1 {
```

Comment:
If there is just 1 servers selected and it is DIP servers but not the one which is bought by user - user will get failure on connect later?

Reply:
yes, user will get later the connection error when also the subscription is checked + the DIP server associated with the account.